core/src/test/java/org/elasticsearch/index/query/SpanNearQueryBuilderTest.java

```diff
@@ -63,7 +63,11 @@ public void testValidate() {
         int clauses = randomIntBetween(1, 10);
         for (int i = 0; i < clauses; i++) {
             if (randomBoolean()) {
-                queryBuilder.clause(new SpanTermQueryBuilder("", "test"));
+                if (randomBoolean()) {
+                    queryBuilder.clause(new SpanTermQueryBuilder("", "test"));
+                } else {
+                    queryBuilder.clause(null);
```

Comment:
Those are two different cases: either clause is null or the inner query is not valid. Would like to test both here.